pkg/devenvironment/name.go

```diff
@@ -82,12 +82,12 @@ func (n NameInferer) InferNameFromDevEnvsAndRepository(ctx context.Context, repo
 		optsRepo := repository.NewRepository(repoURL)
 		cmapRepo := repository.NewRepository(repo)
 		if !optsRepo.IsEqual(cmapRepo) {
-			oktetoLog.Infof("configmap %s with repo %s doesn't match with found repo %s", cmap.Name, repo, repoURL)
+			oktetoLog.Infof("configmap %s with repo %s doesn't match with found repo %s", cmap.Name, cmapRepo.GetAnonymizedRepo(), repoURL)
```

Comment:
I was thinking more about something like:
```
oktetoLog.Infof("configmap %s doesn't match with found repo %s", cmap, repoURL)
```
and configmap would have a `String()` method that shows what's needed, for example, the configmap name, repo URL and okteto manifest path, without credentials.
But take it as a suggestion, whatever works better for you